docs/linting-guide.md

```diff
@@ -0,0 +1,40 @@
+# Linting, Formatting & Other pre-commit Checks
+
+Make sure that you have followed steps in [Getting Started](../CONTRIBUTING.md#getting-started) Section
+
```

Comment:
Include the steps for installing "pre-commit".

```
## Installation

## Usage
```

Reply:
done